core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCRealmConstants.java

```diff
@@ -73,11 +73,21 @@
     public static final String ON_DELETE_USER_DELETE_PERMISSION = "OnDeleteUserRemovePermissionsSQL";
     public static final String IS_DOMAIN_EXISTING = "IsDomainExistingSQL";
     public static final String ADD_DOMAIN = "AddDomainSQL";
+    public static final String APPLICATION_DOMAIN = "Application";
     public static final String UPDATE_USER_PASSWORD = "UpdateUserPasswordSQL";
     public static final String UPDATE_ROLE_NAME = "UpdateRoleNameSQL";
     public static final String ADD_USER_PROPERTY = "AddUserPropertySQL";
     public static final String UPDATE_USER_PROPERTY = "UpdateUserPropertySQL";
     public static final String DELETE_USER_PROPERTY = "DeleteUserPropertySQL";
+    public static final String COUNT_INTERNAL_ONLY_ROLES_SQL = "SELECT COUNT(UM_ID) AS RESULT FROM UM_HYBRID_ROLE WHERE " +
```

Comment:
Move SQL queries to the bottom of the class and keep the PROPERTY variables in the top. Don't mix them.